### ISSUE TITLE
Bug -Text input group - missing documentation section

### ISSUE THREAD

**Issue by srambach** [patternfly/patternfly#6709]
**Describe the problem**
This is an issue with 
- [x] Patternfly 5  
- [x] Patternfly 6  
- [ ] other

The documentation (usage and any accessibility information) is missing from the text input group page.
https://www.patternfly.org/components/text-input-group
